recipes/napari-annotatorj/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "napari-annotatorj" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-annotatorj-{{ version }}.tar.gz
+  sha256: 6fd89e87aa115960ed25041c31f917f2180bffd2f20c35560c781338b1dbd178
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - imagecodecs
+    - keras
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - opencv >=4.5.5
+    - python >=3.7
+    - roifile
+    - scikit-image
+    - tensorflow >=2.5.0
+    - tifffile
+    - qtpy
+    - pyqt
+
+test:
+  imports:
+    - napari_annotatorj
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/spreka/napari-annotatorj
+  summary: The napari adaptation of the ImageJ/Fiji plugin AnnotatorJ for easy image annotation.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - 42864288
```

```suggestion
    - spreka
```

I don't know how this happened, but you probably wanted your name here

(I know review not requested yet, but I have a general interest in napari and curiously took a look when I saw the package title 😄 )